### PR TITLE
Hex Centre Point logic bug fix

### PR DESCRIPTION
**Pull request by pmckeown**
I've been using your library to build a SpriteKit game.  Firstly, thanks heaps for implementing the hex math content that Red Blob Games have posted.  I really appreciate it!

I have found a fixed a small bug with the calculation for determining the centre point of a cell.
As I was laying out hexes onto a screen, I found that the centre point of each rendered hex was twice the value I expected.  You can see from the attached files the difference in rendering.

I also took the opportunity to fix a few typos :)

Thanks again!

Before:
before

After:
after

### CODE REVIEW

File: Sources/HexGrid/Convertor.swift [hunk 1]
```diff
@@ -77,8 +77,8 @@ internal struct Convertor {
         orientation: Orientation) -> Point {
         let axialCoords = coordinates.toAxial()
         let orientation = OrientationMatrix(orientation: orientation)
-        let x = ((orientation.f00 * Double(axialCoords.q)) + (orientation.f10 * Double(axialCoords.r))) * hexSize.width
-        let y = ((orientation.f01 * Double(axialCoords.q)) + (orientation.f11 * Double(axialCoords.r))) * hexSize.height
+        let x = ((orientation.f00 * Double(axialCoords.q)) + (orientation.f10 * Double(axialCoords.r))) * (hexSize.width/2)
+        let y = ((orientation.f01 * Double(axialCoords.q)) + (orientation.f11 * Double(axialCoords.r))) * (hexSize.height/2)
```
Comment (pmckeown) [inline Sources/HexGrid/Convertor.swift:82]: This was the only functional change

**Comment by fananek** [comment]
Good catch, thanks for PR.

**Comment by pmckeown** [comment]
No worries!

File: Sources/HexGrid/Convertor.swift (hunk 1 above)
Comment (chessboy) [reply Sources/HexGrid/Convertor.swift:82]: Great fix. I am also using this amazing library with SpriteKit and I thought I was using it wrong so I ended up scaling my node radius by 2 to compensate. Just pulled latest and removed my kludge. Thanks to both of you!

Comment (pmckeown) [reply Sources/HexGrid/Convertor.swift:82]: No worries.  Glad to have contributed!

**Pull request merged**